src/main/java/duke/task/Deadlines.java

```diff
@@ -0,0 +1,74 @@
+package duke.task;
+
+import duke.utils.Parser;
+
+import java.util.Date;
+
+/**
+ * Deadline object extending from ExecutableTask
+ */
+public class Deadlines extends ExecutableTask {
```

Comment:
I think the name of the class should not be of a plural form according to the Java coding standard, because plural forms are usually used for a collection. If you want to create a collection of Deadlines object, then it would be quite hard to distinguish the two.